contributors/guide/style-guide.md

```diff
@@ -0,0 +1,672 @@
+    unclear if the period is a part of the previous object or word.
+* Add a single space after a period when beginning a new sentence.
+* Avoid usage of exclamation points unless they are a part of a code example.
+* Use an [Oxford comma] when a list contains 3 or more elements.
+
+**[Quotation:](#quotes)**
+* Use double-quotation marks (`" "`) over single-quotation marks (`' '`).
+  * **Exception:** In code snippets where quotation marks have specific meaning.
+  * **Exception:** When nesting quotation marks inside another set of quotation
+    marks.
+* Punctuation should be outside of quotation marks following the international
+  (British) standard.
+
+
+### Cheatsheet: Markdown
+
+**[Code blocks:](#code-blocks)**
+* When possible, reference the language at the beginning of a Code Block.
+* When a code block is used to reference a shell, do not include the command
+  prompt (`$`).
+  * **Exception:** When a code block is used to display raw shell output.
+* Separate commands from output.
+
+**[Emphasis:](#emphasis)**
+* Use two asterisks (`**`) for **Bold** text.
+* Use an underscore (`_`) for _Italics_.
+* Use two tildes (`~~`) for ~~Strikethrough~~.
+ 
+**[Headings:](#headings)**
+* Use a single `H1` (`#`) Heading per document.
+  * **Exception:** `H1` may be used multiple times in the same document when
+    there is a large content shift or "chapter" change.
+* Follow the Header hierarchy of `H2` > `H3` > `H4` > `H5` > `H6`.
+* Use sentence-style capitalization in titles (first word and proper nouns).
+* Avoid using special characters.
+* Leave exactly 1 new line after a heading.
+* Avoid using links in headings.
+
+**[Horizontal rules:](#horizontal-lines)**
+* Use three dashes (`---`) to denote a horizontal rule.
+* Use a horizontal rule (`---`) to logically separate large sections.
+
+**[Line length:](#line-length)**
+* Prefer an 80 character line limit.
+
+**[Links:](#links)**
+* Prefer using reference style links over inline style links.
+* When linking within the same directory, use a relative link.
+* When linking to a document outside of the current directory, use the absolute
+  path from the root of the repository.
+* When linking to a file in another Kubernetes github repository, use the
+  `k8s.io` url shortener.
+  * git.k8s.io -> github.com/kubernetes
+  * sigs.k8s.io -> github.com/kubernetes-sigs
+
+**[Lists:](#lists)**
+* Capitalize the first character of each entry unless the item is explicitly
+  case sensitive.
+* End each entry with a period if it is a sentence or phrase.
+* Use a colon (`:`) to separate a list item name from the explanatory text.
+* Leave a blank line after each list.
+* Use `*` for unordered lists.
+* For ordered lists repeating `1.` may be used.
+* When inserting a code block into an ordered list, indent (space) an additional
+  two times.
+
+**[Metadata:](metadata)**
+* Include a yaml metadata header at the beginning of the document.
+* Metadata must include the `title` attribute.
```

Comment:
The metadata is intended for use with the contributor site. Maybe rephrase to something like "If the document is to be surfaced on the contributor-site, include a metadata header"?

Reply:
I mean, if we think everything here should have a metadata header because it's all in some way destined for that site, that's fine too.  This just raised my eyebrows as a new ideal state rather than something that describes how we mostly do things

Reply:
I'd say flip to conditional for now. If we wind up in the situation where most of the content is being surfaced, the guideline can be updated to become the default.